mne/decoding/transformer.py

```diff
@@ -286,7 +296,7 @@ def __init__(self, sfreq=2 * np.pi, fmin=0, fmax=np.inf, bandwidth=None,
         self.verbose = verbose
         self.normalization = normalization
 
-    def fit(self, epochs_data, y):
+    def fit(self, epochs_data, y=None):
```

Comment:
Good point, I will update the docstring if this change remains. I made it at the beginning of the PR because I thought it was a bug. I was getting some error when calling "fit" on my data because I wasn't supplying the epoch types (`y`) along with it. `y` isn't used at all in the function, and it's a kwarg in the `transform` method, so I thought it was a mistake and should be an optional kwarg rather than a positional argument. Maybe this isn't the case?